.storybook/main.js

```diff
@@ -87,7 +81,4 @@ module.exports = {
       },
     },
   },
-  docs: {
-    autodocs: true,
-  },
```

Comment:
Remove the auto-generated docs for each story set, and only add them as needed.

Can be done in a separate PR if desired